Flow.Launcher.Infrastructure/Image/ImageCache.cs

```diff
@@ -2,17 +2,18 @@
 using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading.Tasks;
 using System.Windows.Media;
 
 namespace Flow.Launcher.Infrastructure.Image
 {
     [Serializable]
     public class ImageCache
     {
-        private const int MaxCached = 5000;
+        private const int MaxCached = 50;
         public ConcurrentDictionary<string, int> Usage = new ConcurrentDictionary<string, int>();
         private readonly ConcurrentDictionary<string, ImageSource> _data = new ConcurrentDictionary<string, ImageSource>();
```

Comment:
Maybe we should merge this two dictionary to one by creating another utility class, which will allow us to remove element easier?

Reply:
yeah, let me have a look

Reply:
I have finished that, if you want, I can add that to the pr.

Reply:
haha ok cool, yes please